docs/WHO_IS_USING_USWDS.md

```diff
@@ -21,6 +21,7 @@ Below are a list of websites and applications currently using the U.S. Web Desig
 - [Be Tobacco Free - HHS](https://betobaccofree.hhs.gov/)
 - [The Centers for Medicare and Medicaid Services (CMS) Blue Button API Docs](https://bluebutton.cms.gov/developers/)
 - [Centers of Excellence (GSA)](https://coe.gsa.gov/)
+- [Challenge.Gov](https://www.challenge.gov/)
 - [Christmas Tree Permits - U.S. Forest Service](https://openforest.fs.usda.gov/christmas-trees/forests)
```

Comment:
@mhotch24 do you know where this Christmas tree permits link came from? Not seeing it in the current doc. The link appears to be broken too.

https://github.com/uswds/uswds/blob/develop/docs/WHO_IS_USING_USWDS.md